src/main/java/me/karakelley/http/controllers/RedirectController.java

```diff
@@ -0,0 +1,47 @@
+package me.karakelley.http.controllers;
+
+import me.karakelley.http.Request;
+import me.karakelley.http.Response;
+import me.karakelley.http.Status;
+
+public class RedirectController implements Controller {
+
+  private final Request request;
+
+  public RedirectController(Request request) {
+    this.request = request;
+  }
+
+  public Response respond() {
+    Response response = new Response();
+
+    if (request.validRequestLine() && request.getMethod().equals("GET")) {
+      setResponse(response);
+    } else {
+      response.setStatus(Status.NOT_FOUND);
+    }
+    return response;
+  }
+
+  public Response setResponse(Response response) {
+    response.setStatus(Status.MOVED_PERMANENTLY);
+    response.setHeaders("Location", "http://" + findHostAndPort() + "/");
+    return response;
+  }
+
+  public String findHostAndPort() {
```

Comment:
Does this need to be public? It's not in the `Controller` interface, so anything calling this would have to know it has an instance of this concrete class